wiki/People/Beatmap_Nominators/id.md

```diff
@@ -0,0 +1,172 @@
+[false]: /wiki/shared/false.png "No"
+[true]: /wiki/shared/true.png "Yes"
+
+Beatmap Nominator
+==================
+
+*Lihat juga: [Peraturan Beatmap Nominator](/wiki/Beatmap_Nominator_Rules) dan [Informasi Umum untuk Beatmap Nominator](/wiki/Beatmap_Nominator_General_Information).*
+
+Beatmap Nominator, dikenal juga dengan *BN*, adalah kelompok yang bertanggung jawab atas [penominasian beatmap](/wiki/Submissions/Beatmap_Ranking_Procedure). Sejak 2014-08-22, Beatmap Nominator telah dipisahkan dari bagian staff, perubahan ini meliputi pencabutan hak moderasi di chat in-game, nama merah di in-game, dan di dalam forum. Pada 2015-02-01, nama kelompok berubah dari *Beatmap Appreciation Team* menjadi *Beatmap Nominators*. Saat ini, mereka memiliki warna ungu pada nama mereka di forum.
+
+Peran dan Tanggung Jawab
+--------------------------
+
+Anggota Beatmap Nominator bertanggung jawab terutama untuk [modding](/wiki/Modding) [beatmaps berstatus pending](/wiki/Beatmaps) ke dalam ranah status ranking. Beatmap Nominator adalah orang-orang yang dapat dimintai saran tentang mapping atau modding.
+
+Menjadi seorang Beatmap Nominator
+----------------------------
+
+Dengan perombakan team pada Agustus 2014 lalu, bergabung menjadi Beatmap Nominator jauh lebih mudah dibandingkan dengan yang dulu. [Quality Assurance Team](https://osu.ppy.sh/g/7) akan mengumumkan pendaftaran Beatmap Nominator berupa post di [Pending Beatmaps Subforum](https://osu.ppy.sh/forum/6) dalam kurun waktu 1 atau 2 bulan.
```

Comment:
>berupa post di [Pending Beatmaps Subforum](https://osu.ppy.sh/forum/6) dalam kurun waktu 1 atau 2 bulan

Try `dalam bentuk` instead of `berupa`, and `tenggang` or `rentang` instead of `kurun`?